packages/react-examples/.storybook/preview-loader.js

```diff
@@ -20,19 +19,5 @@ export default function loader(source) {
     return;
   }
 
-  source = source.replace(/PACKAGE_NAME/g, packageName);
-
-  if (packageName === 'react') {
```

Comment:
how come that this can be removed ?

Reply:
It's sort of in the description but I maybe could have explained more clearly.

> Get rid of the ability to dynamically load stories for dependencies (REACT_DEPS replacement) and hardcode loading from just react-focus instead (it turns out this had been broken for awhile)

That feature/hack was more relevant during the mostly-abandoned effort to split `@fluentui/react` ("mostly" because the react-focus package survived, for historical reasons which actually no longer make sense/are necessary), or when we had the converged component stories in here and needed to roll them up into the react-components storybook.